apps/connect/src/env/token-bridge.mainnet.ts

```diff
@@ -348,6 +348,244 @@ export const ENV: Env = {
+          coinGeckoId: 'yaku',
+          decimals: {
+            default: 9,
+          },
+        },
+        Yakuarbitrum: {
+          key: 'Yakuarbitrum',
+          symbol: 'Yaku',
+          nativeChain: 'arbitrum',
+          tokenId: {
+            chain: 'arbitrum',
+            address: '0xB00eaEDB98F1e30ad545703d8Ff14b24D109514f',
+          },
+          icon: 'https://assets.coingecko.com/coins/images/26785/standard/yaku_logo.png?1706636424',
+          coinGeckoId: 'yaku',
+          decimals: {
+            default: 9,
+          },
+        },
+        Yakuavalanche: {
+          key: 'Yakuavalanche',
+          symbol: 'Yaku',
+          nativeChain: 'avalanche',
+          tokenId: {
+            chain: 'avalanche',
+            address: '0xB00eaEDB98F1e30ad545703d8Ff14b24D109514f',
+          },
+          icon: 'https://assets.coingecko.com/coins/images/26785/standard/yaku_logo.png?1706636424',
+          coinGeckoId: 'yaku',
+          decimals: {
+            default: 9,
+          },
+        },
+        WeatherXMethereum: {
+          key: 'WeatherXMethereum',
+          symbol: 'WXM',
+          nativeChain: 'ethereum',
+          tokenId: {
+            chain: 'ethereum',
+            address: '0xde654f497a563dd7a121c176a125dd2f11f13a83',
+          },
+          icon: 'https://assets.coingecko.com/coins/images/38154/standard/weatherxm-network-logo.png?1716668976',
+          coinGeckoId: 'weatherxm-network',
+          decimals: {
+            default: 18,
+          },
+        },
+        WeatherXMsolana: {
+          key: 'WeatherXMsolana',
+          symbol: 'WXM',
+          nativeChain: 'solana',
+          tokenId: {
+            chain: 'solana',
+            address: 'wxmJYe17a2oGJZJ1wDe6ZyRKUKmrLj2pJsavEdTVhPP',
+          },
+          icon: 'https://assets.coingecko.com/coins/images/38154/standard/weatherxm-network-logo.png?1716668976',
+          coinGeckoId: 'weatherxm-network',
+          decimals: {
+            default: 9,
+          },
+        },
+        JitoSOLsolana: {
+          key: 'JitoSOLsolana',
+          symbol: 'JITOSOL',
+          nativeChain: 'solana',
+          tokenId: {
+            chain: 'solana',
+            address: 'J1toso1uCk3RLmjorhTtrVwY9HJ7X8V9yYac6Y7kGCPn',
+          },
+          icon: 'https://assets.coingecko.com/coins/images/28046/standard/JitoSOL-200.png?1696527060',
+          coinGeckoId: 'jito-staked-sol',
+          decimals: {
+            default: 9,
+          },
+        },
+        JitoSOLarbitrum: {
+          key: 'JitoSOLarbitrum',
+          symbol: 'JITOSOL',
+          nativeChain: 'arbitrum',
+          tokenId: {
+            chain: 'arbitrum',
+            address: '0x83e1d2310Ade410676B1733d16e89f91822FD5c3',
+          },
+          icon: 'https://assets.coingecko.com/coins/images/28046/standard/JitoSOL-200.png?1696527060',
+          coinGeckoId: 'jito-staked-sol',
+          decimals: {
+            default: 18,
```

Comment:
this one should be 9
https://arbiscan.io/token/0x83e1d2310ade410676b1733d16e89f91822fd5c3